src/pages/home/_hero.tsx

```diff
@@ -0,0 +1,226 @@
+import React, { ReactNode } from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import VerticalCarousel from './_vertical-carousel.js'
+import Slideshow from 'components/elements/slideshow'
+import device from 'themes/device'
+import { LinkButton } from 'components/form'
+import { Container, Box, Flex } from 'components/containers'
+import { BackgroundImage, Header } from 'components/elements'
+import { Localize, localize } from 'components/localization'
+
+const query = graphql`
+    query {
+        hero_background: file(relativePath: { eq: "home/hero_bg.png" }) {
+            ...homePageHeroFadeIn
+        }
+        hero_platform1: file(relativePath: { eq: "home/hero_platform1.png" }) {
+            ...fadeIn
+        }
+        hero_platform2: file(relativePath: { eq: "home/hero_platform2.png" }) {
+            ...fadeIn
+        }
+        hero_platform3: file(relativePath: { eq: "home/hero_platform3.png" }) {
+            ...fadeIn
+        }
+        hero_platform4: file(relativePath: { eq: "home/hero_platform4.png" }) {
+            ...fadeIn
+        }
+    }
+`
+type HeroProps = {
+    is_ppc: boolean
+}
+
+const contents = [
+    <Localize key={0} translate_text="Tight spreads" />,
+    <Localize key={1} translate_text="Sharp prices" />,
+    <Localize key={2} translate_text="24x7 trading" />,
+    <Localize key={3} translate_text="100+ tradeable assets" />,
+    <Localize key={4} translate_text="20+ years of experience" />,
+]
+
+const contents_ppc = [
+    <Localize key={3} translate_text="Tight spreads" />,
+    <Localize key={2} translate_text="Sharp prices" />,
+    <Localize key={1} translate_text="100+ tradeable assets" />,
+    <Localize key={0} translate_text="20+ years of experience" />,
+]
+
+const StyledContainer = styled(Container)`
+    width: 100%;
+    max-width: 100%;
+    margin: 0 auto;
+    padding: 0 12rem;
+
+    @media ${device.laptopM} {
+        padding: 0 8rem;
+    }
+
+    @media ${device.tabletL} {
+        align-items: center;
+        max-width: 588px;
+        padding: 0 16px;
+    }
+`
+const StyledHeroContainer = styled(Flex)`
+    max-width: 1200px;
+    margin: 0 auto;
+
+    @media ${device.tabletL} {
+        flex-direction: column;
+        max-width: 100%;
+    }
+`
+const StyledHeaderWrapper = styled(Flex)`
+    @media ${device.tabletL} {
+        flex-wrap: wrap;
+        justify-content: flex-start;
+    }
+`
```

Comment:
We can use props and responsive props for these elements. This applies to most of our elements such as `Flex`, `Container`, `Header` & `Box`.